neo.UnitTests/SmartContract/UT_InteropService.cs

```diff
@@ -188,5 +201,814 @@ private void AssertNotification(StackItem stackItem, UInt160 scriptHash, int not
             CollectionAssert.AreEqual(scriptHash.ToArray(), array[0].GetByteArray());
             Assert.AreEqual(notification, array[1].GetBigInteger());
         }
+
+        [TestMethod]
+        public void TestExecutionEngine_GetScriptContainer()
+        {
+            var engine = GetEngine(true);
+            InteropService.Invoke(engine, InteropService.System_ExecutionEngine_GetScriptContainer).Should().BeTrue();
+            engine.CurrentContext.EvaluationStack.Pop().Should().Be(StackItem.FromInterface(engine.ScriptContainer));
+        }
+
+        [TestMethod]
+        public void TestExecutionEngine_GetExecutingScriptHash()
+        {
+            var engine = GetEngine();
+            InteropService.Invoke(engine, InteropService.System_ExecutionEngine_GetExecutingScriptHash).Should().BeTrue(); ;
+            engine.CurrentContext.EvaluationStack.Pop().GetByteArray().ToHexString()
+                .Should().Be(engine.CurrentScriptHash.ToArray().ToHexString());
```

Comment:
I think this test is the same original code... in this sense, it would be better to have a practical example here, e.g., create a craft script, execute it, and get executingScriptHash.

Reply:
Creating practical example is more complex indeed, and I think it is enough using `Invoke` way to cover the whole line of the target method for unit test. For the further senario test, we should use the practical example to test the real senario case.